build/tf-wrapper.sh

```diff
@@ -139,7 +139,7 @@ convert_path() {
 ## Terraform apply for single environment.
 tf_apply() {
   local path=$1
-  local tf_env="${path#$base_dir/}"
+  local tf_env="${path#"$base_dir"/}"
```

Comment:
These are good - usually the pr lint flags these - I have had to wrap all vars before in other repos

Reply:
Thanks
Seems the linter has been `Expected — Waiting for status to be reported` for a number of weeks (possibly waiting for an approval to run). I validated locally using shellcheck. If this is likely to get flagged by the linter I can change it back as it is not functional change that is part of the PR. 

For reference here is the issue flagged by shellcheck: https://www.shellcheck.net/wiki/SC2295

Reply:
/gcbrun to get the linter going